bbs/lib/mainfunc.php

```diff
@@ -1,11 +1,11 @@
 <?php
+require_once "../../config.php";
 function mainfunc($posts,$debug=false){
 	$ip=$_SERVER["REMOTE_ADDR"];
 	@$token=$_COOKIE['token'];
-	$url="http://127.0.0.1/api/jiekouapi.php?ip=$ip&token=$token";
+	$url=CAPU_JIEKOUAPI."?ip=$ip&token=$token";
```

Comment:
I have add some into Conventions.
Please modify all related changes made by this PR.